src/main/java/seedu/address/model/travelplan/TravelPlan.java

```diff
@@ -90,6 +92,14 @@ public static boolean isValidStartAndEndDate(WanderlustDate startDate, Wanderlus
         return startDate.getValue().compareTo(endDate.getValue()) <= 0;
     }
 
+    /**
+     * Returns true if the start date is set on today's date or after today's date. Prevent any date inputs that are
+     * too old (Etc, 1111-11-11)
+     */
+    public static boolean isStartDateAfterToday(WanderlustDate startDate) {
+        return startDate.getValue().compareTo(LocalDate.now()) >= 0;
```

Comment:
All these getValue() haha, kind of wanna abstract all of them out XD, but see you want or not. I think its ok? Like add a method in WanderlustDate compareTo() so can just call startDate.compareTo() instead of this, need getValue() everytime.

Reply:
fixed